cmd/helm/installer/uninstall.go

```diff
@@ -21,13 +21,12 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	"k8s.io/helm/pkg/kube"
 	"k8s.io/kubernetes/pkg/api"
 	kerrors "k8s.io/kubernetes/pkg/api/errors"
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
-
-	"k8s.io/helm/pkg/kube"
```

Comment:
@adamreese I just ran:
```
$ goimports -w cmd
$ goimports -w ./cmd/..
$ goimports -w ./pkg/..
$ goimports -w pkg
```
Still no change. In fact that make changes some other unrelated files:
```
	modified:   pkg/chartutil/load.go
	modified:   pkg/releaseutil/filter_test.go
	modified:   pkg/releaseutil/sorter_test.go
```
So, I skipped those.